Don't remove -fPIC for static extensions.  The only one we compile
statically is DynaLoader which may be linked to shared objects like
mod_perl.

diff -ur perl-5.6.1.orig/ext/util/make_ext perl-5.6.1/ext/util/make_ext
--- perl-5.6.1.orig/ext/util/make_ext	Fri Feb 23 13:57:54 2001
+++ perl-5.6.1/ext/util/make_ext	Wed May 23 22:51:22 2001
@@ -88,7 +88,7 @@
 # check link type and do any preliminaries
 case "$target" in
 	    # convert 'static' or 'dynamic' into 'all LINKTYPE=XXX'
-static)     makeargs="LINKTYPE=static CCCDLFLAGS="
+static)     makeargs="LINKTYPE=static"
 	    target=all
 	    ;;
 dynamic)    makeargs="LINKTYPE=dynamic";
